src/AppKernel.php

```diff
@@ -68,27 +67,27 @@ public function registerBundles() : array
         return $bundles;
     }
 
-    public function getName() : string
```

Comment:
Doing a warmup generates code that extends this class, causing a fatal error. Same reason it's not `final`. 😒